apps/api/src/domain/games/memory-match/queries/memory-match.controller.ts

```diff
@@ -19,11 +23,28 @@ import { MemoryMatchService } from '../services/memory-match.service';
 export class MemoryMatchController {
     constructor(private readonly memoryMatchService: MemoryMatchService) {}
 
+    // TODO this should be a MemoryMatchRoundViewModel
+    @ApiResponse({
+        status: HttpStatusCode.ok,
+        type: MemoryMatchRound,
+        description: 'returns a single memory match round',
+        example: buildTestInstance(MemoryMatchRound),
```

Comment:
What we really want is a helper called `getCoscradDataExample(MemorMatchRoundViewModel)`. Add a TODO for this and we can do that on a near-future branch.